### ISSUE TITLE
Delayed Account Creation: inform the users that the feature is only available with block themes

### ISSUE THREAD

**Issue by pmcpinto** [woothemes/woocommerce#52989]
**Description:** currently, we don't inform users that the After checkout feature is only available with the order confirmation template (block themes). We know that some users start with a block theme and then transition to other solutions. This unawareness can create problems in situations where users enable the After Checkout feature with the expectation that it will work in every situation, but the account creation settings will change if the store transitions to a non-block theme. 

We need a sound solution for this before enabling this feature as default for new users. 

**Acceptance criteria:**

- [ ] Explore how to highlight this information on the settings page.
- [ ] Explore how to highlight this information on the order confirmation template editor. This can be extended to any order confirmation change, as we currently don't inform users that it's only available with block themes.
- [ ] Assess if we should surface this information in other surface areas (example: send an Inbox message when users switch to a non-block theme).

**Comment by elizaan36** [comment]
> Explore how to highlight this information on the settings page.

Could we detect the theme and present a tooltip on the disabled selection?

![Image](https://github.com/user-attachments/assets/7cab637b-8347-4415-a3aa-3dd5d19b8b22)

> Explore how to highlight this information on the order confirmation template editor. This can be extended to any order confirmation change, as we currently don't inform users that it's only available with block themes.

I think I need to understand the issue here. If the user is on the template editor, aren't they already on a block theme? I think seeing the feature available or not in the canvas is enough of an indication. Or what is the use case in the template editor?

**Comment by pmcpinto** [comment]
> Could we detect the theme and present a tooltip on the disabled selection?

AFAIK this is doable. @mikejolley what do you think?

Elizabeth, do you think we should show this information in other surface areas?

> I think I need to understand the issue here. If the user is on the template editor, aren't they already on a block theme? I think seeing the feature available or not in the canvas is enough of an indication. Or what is the use case in the template editor?

Prevention. We don't inform users that any Order Confirmation template changes aren't available if they switch to a non-block theme.

**Comment by mikejolley** [comment]
@pmcpinto @elizaan36 yes we can include (another) tooltip with different wording. Its not much of a sell though—how does a merchant not using a block theme understand what they are missing?

Technical side yes we can add a tooltip. It would be nice to present the benefits of switching here/somewhere else which illustrates why they should switch. This applies to order confirmation page also.

**Comment by elizaan36** [comment]
I think the feature itself is a great sell! You can only have this feature if you use Checkout blocks is a super clear message.

> Prevention. We don't inform users that any Order Confirmation template changes aren't available if they switch to a non-block theme.

Adding messaging in the Editor isn't necessary imo. I'm concerned that adding more warnings into the Editor about features that are/aren't available will be noisy and detract from the experience.

> Elizabeth, do you think we should show this information in other surface areas?

As long as the feature announcements go out via Marketing and dev blogs and there's awareness of the value of the feature to improve checkout conversion that should be enough! An Inbox note would be an option to cover all the bases but I'm not convinced that would have enough of an impact.

**Comment by mikejolley** [comment]
> I think the feature itself is a great sell! You can only have this feature if you use Checkout blocks is a super clear message.

@elizaan36 The feature is, but the message is hidden in a tooltip and is just text. How does the merchant know what the feature does?

**Comment by pmcpinto** [comment]
> Adding messaging in the Editor isn't necessary imo. I'm concerned that adding more warnings into the Editor about features that are/aren't available will be noisy and detract from the experience.

Makes sense. We can always revisit this if we get feedback from users.

> As long as the feature announcements go out via Marketing and dev blogs and there's awareness of the value of the feature to improve checkout conversion that should be enough! An Inbox note would be an option to cover all the bases but I'm not convinced that would have enough of an impact.

My concern is that the Marketing and Dev blog channels don't reach all Woo users. Complementing that with admin messaging could expand the reach.

> @elizaan36 The feature is, but the message is hidden in a tooltip and is just text. How does the merchant know what the feature does?

That's a good point. Maybe we should include a paragraph with additional context and a link to the documentation section about the new feature.

**Comment by elizaan36** [comment]
We do have a nice new component that we could use for this. What do y'all think about using the spotlight component? This could be used in addition to the tooltip.

Then an Inbox note could link directly there to get more visibility since this settings page is quite obscure. The Learn more link could go to the docs (is there a docs page for this?).

<img width="1492" alt="Image" src="https://github.com/user-attachments/assets/01779e45-0242-4557-ae3b-288344058cc3" />

We would want to be thoughtful about the audience that sees the inbox note. We wouldn't want to present it to users whose stores were set up with the default, for example.

**Comment by ralucaStan** [comment]
@elizaan36 is there a screen where we can see this component? Is this built for classic admin views?

**Comment by elizaan36** [comment]
Yes I believe it was built originally in WooCommerce admin to bring attention to the new Coming Soon/Live mode.

I know you can see it now in WooPayments to call out the name change from Deposits to Payouts.

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/21366e09-bb23-4382-b73b-3270fbf6c4af" />

**Comment by pierorocca** [comment]
>  Its not much of a sell though—how does a merchant not using a block theme understand what they are missing?

@mikejolley is spot on and keeping features visible, but disabled, is a great way to build FOMO. How might we use chips or badges to denote a "Blocks Feature"? Picture several of these high value add feature badges sprinkled throughout the admin, and over time more and more becoming present and noticed. It visually shows, without being hidden in a tooltip, that the cool stuff is happening on Blocks. The multiple occurences of the badges with the term "Blocks" helps it become part of the vernacular for the unaware. If the merchant switches over to Blocks the badges are removed to eliminate the clutter.

Thoughts?

**Comment by samrat-biswas-git** [comment]
How we communicate the block-theme requirement for Delayed Account Creation:

Settings page (primary surface):
- Keep the feature visible but disabled on non-block themes (rather than hiding it) — this builds awareness and FOMO instead of silently dropping the option.
- Add a tooltip on the disabled control explaining it requires the Checkout blocks / a block theme — framed around the benefit (what the feature does and why it's worth having), not just the limitation.
- Add a "Blocks feature" badge/chip so the requirement is clear at a glance.

Discoverability:
- Use the spotlight component to draw attention to the setting, since the page is easy to miss.
- Add an Inbox note pointing to the setting, with a "Learn more" link.

Editor:
- No additional warnings in the editor — we agreed that would be noisy and detract from the experience. We'll revisit only if user feedback shows it's needed.

Next step: design to mock up the disabled-state tooltip, the badge/chip, and the spotlight/Inbox treatment; then we'll break down the engineering work. Keeping needs: design.

**Comment by elizaan36** [comment]
Thanks for the clear summary of next steps @samrat-biswas-git . I'll share a simplified design proposal for review.

As a recap, when the store is on a classic theme/classic checkout, the "After checkout" option doesn't appear at all. The goal of this work is to present block checkout features in such a way that would incentivize merchants to make the switch.

### Designs

<img width="1267" height="515" alt="Image" src="https://github.com/user-attachments/assets/9c02f402-b61d-4298-a226-86e485f82e32" />

<img width="1267" height="515" alt="Image" src="https://github.com/user-attachments/assets/693f4faf-b800-4aa2-be56-03b4f4d12519" />

### Details
- Present the "After checkout" option to all stores and detect when the store is using classic checkout (so for the stores who currently don't have this option at all), we'd present this solution - the option is unavailable, but promotes the feature and links to the docs to learn more.
- Use a badge alongside the option for `Requires block checkout`. This is a reusable badge (as discussed in the thread) to promote block checkout features to classic checkout users in the admin going forward. Local pickup is another one that comes to mind.
- Use body copy for the primary explanation and to describe the value prop of making the switch.

> Gives shoppers a smoother way to create an account after placing
an order. Available with a block theme and block checkout. [Learn more](https://woocommerce.com/document/woocommerce-store-editing/customizing-cart-and-checkout/)

- Use a tooltip on the disabled checkbox as a short reinforcement:

> Your store is using classic checkout. Use the block checkout to enable this option.

No other settings are changed. IMO the spotlight and inbox note aren't necessary for this pass, we can consider more aggressive promotion as needed.

Figma: xsRWL4dMRYrTsMrLbXIuoA-fi-1437-35413&t=V7BeAYcskankD6eh-1

**Comment by samrat-biswas-git** [comment]
This looks great, @elizaan36. The badge + tooltip + value-prop copy cover the goal well.

Agree on building the "Requires block checkout" badge as a reusable component, since it'll apply to other block-checkout features (Local Pickup, etc.) as you flagged. 

Moving it as "ready for engineering"
